psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -2027,6 +2027,17 @@ std::tuple<SharedMatrix, SharedMatrix, SharedMatrix, SharedMatrix> PopulationAna
 
     if (!is_converged) throw ConvergenceError<int>("MBIS", max_iter, conv, delta_rho_max_0, __FILE__, __LINE__);
 
+    // Final population and width
+    if (print_output && debug >= 1) {
+        for (int atom = 0; atom < num_atoms; atom++) {
+            for (int m = 0; m < mA[atom]; m++) {
+                outfile->Printf("  FINAL ATOM %d, SHELL %d, POP %8.5f, WIDTH %8.5f\n", atom + 1, m + 1, Nai[atom][m],
```

Comment:
Would it be helpful to also print out the angular momentum of the shell (L)?

Reply:
Hi @andyj10224, in MBIS the shells are just Slater functions, so they are spherically symmetric (L=0).